### PR TITLE
OTG Tests Updates - RT-5.2, RT-5.3, TE-14.1

### LINKED ISSUE #558: OTG: RT-5.2: Aggregate Interfaces
- [README](https://github.com/openconfig/featureprofiles/blob/main/feature/interface/aggregate/otg_tests/aggregate_test/README.md)

### LINKED ISSUE #559: OTG: RT-5.3: Aggregate Balancing
- [README](https://github.com/openconfig/featureprofiles/blob/main/feature/interface/aggregate/otg_tests/balancing_test/README.md)

### LINKED ISSUE #577: OTG: TE-14.1: gRIBI Scaling

### PR DESCRIPTION
**Pull request by octpetre**
- Updated TE-14.1 to match the ATE version
- Removed deprecated "SetPortName" option from the otg device configuration
- Fixes #558
- Fixes #559
- Fixes #577

### CODE REVIEW

**Comment by octpetre** [comment]
@michaelore could you please approve this?

**Comment by octpetre** [comment]
@jasdeep-hundal could you please approve?

File: feature/interface/aggregate/otg_tests/balancing_test/balancing_test.go [hunk 1]
```diff
@@ -260,14 +260,18 @@ func (tc *testCase) verifyLAG(t *testing.T) {
 		t.Logf("Waiting LAG OTG ports to start collecting and distributing")
 		for _, p := range tc.atePorts[1:] {
 			_, ok := gnmi.Watch(t, tc.ate.OTG(), gnmi.OTG().Lacp().LagMember(p.ID()).Collecting().State(), time.Minute, func(val *ygnmi.Value[bool]) bool {
+				time.Sleep(1 * time.Second)
```
Comment (marcushines) [inline feature/interface/aggregate/otg_tests/balancing_test/balancing_test.go:263]: why is this here?

Await is edge triggered - waiting a second shouldn't provide any value

File: feature/interface/aggregate/otg_tests/balancing_test/balancing_test.go [hunk 2]
```diff
@@ -260,14 +260,18 @@ func (tc *testCase) verifyLAG(t *testing.T) {
...
 				col, present := val.Val()
+				t.Logf("collecting for port %v is %v and present is %v", p.ID(), col, present)
 				return present && col
 			}).Await(t)
 			if !ok {
 				t.Fatalf("OTG LAG port %v is not collecting", p)
 			}
 			_, ok = gnmi.Watch(t, tc.ate.OTG(), gnmi.OTG().Lacp().LagMember(p.ID()).Distributing().State(), time.Minute, func(val *ygnmi.Value[bool]) bool {
+				time.Sleep(1 * time.Second)
```
Comment (marcushines) [inline feature/interface/aggregate/otg_tests/balancing_test/balancing_test.go:272]: same here

File: feature/gribi/otg_tests/gribi_scaling/gribi_scaling_test.go [hunk 3]
```diff
@@ -383,7 +383,6 @@ func configureATE(t *testing.T, top gosnappi.Config, ate *ondatra.ATEDevice, ate
 
 	dev := top.Devices().Add().SetName(Name + ".Dev")
 	eth := dev.Ethernets().Add().SetName(Name + ".Eth")
-	eth.Connection().SetChoice("port_name")
```
Comment (marcushines) [inline feature/gribi/otg_tests/gribi_scaling/gribi_scaling_test.go:386]: just curious on this one why is this removed?

Comment (octpetre) [reply feature/gribi/otg_tests/gribi_scaling/gribi_scaling_test.go:386]: Basically "eth.SetPortName" is deprecated and it was conflicting with eth.Connection.SetChoice....
Now we have to correct way.

**Review by jasdeep-hundal — APPROVED** [review]
